.travis.yml

```diff
@@ -8,17 +8,17 @@ osx_image: xcode7
 # Each will be run as an independent, hermetic test (in parallel when resources permit)
 # on Traivs.
 env:
-  - TEST_DIR=com.google.code.gson-gson
-  - TEST_DIR=org.joda-joda-convert
-  - TEST_DIR=joda-time-joda-time
-  - TEST_DIR=org.joda-joda-primitives
-  - TEST_DIR=org.apache.commons-commons-lang3
+  - TEST_DIR=libraryBuilds/com.google.code.gson-gson
```

Comment:
The constant prefix here obfuscates what is important: the varying name of the library.
Please append the prefix where it is needed later on.
For long test names it also makes it harder to see on the Travis multi-build screen.